src/components/BookSeaat.js

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState, useContext } from "react";
+import { GlobalContext } from "../context/GlobalState";
+import { SeatBook, SeatContainer } from "../styles/Search.styled";
+import { ConfirmBtn, MainBtn } from "../styles/Modal.styled";
+import { useParams,useLocation } from "react-router-dom";
+import ConfirmBook from "./ConfirmBook";
+import Alphasec from "./Alphasec";
+import SeatNumbers from "./SeatNumbers";
+import { data } from "../data.js";
+import SeatIcon from "./Seats";
+import BlueSvg from "./Blue";
+import Desable from "./Desable";
+
+
+const BookSeaat = () => {
+  const [seatbook, setSeatBook] = useState(data);
+  const [selectedSeats, setSelectedSeats] = useState([])
+  const[DisplayModal,setDisplayModal] = useState(false)
```

Comment:
use consistent case. For component names, use PascalCase, for everything else use camelCase.